### DISCUSSION TITLE
Repo Update Problem

### DISCUSSION THREAD

**Discussion by Hafiz-Sakib** [community/community#37654] [Repositories] [answered]
Hi.
For last 2 days, my GitHub repository always showing "update 5 hours ago", whether I have updated the repo few seconds ago.

It's really updated and showed inside the repo but outside of the repo, it shows an update 5 hours ago.

**Comment by jfullstackdev** [comment, marked as answer]
what repo is that ? 

seems your repos are just fine 

![image](https://user-images.githubusercontent.com/47092464/198883944-a3f94234-1c0d-405b-880d-8e1897b34a01.png)

**Reply by Hafiz-Sakib** [reply]
I had a manual windows update recently. after that, I forgot to change the timezone. Now the Problem is fixed. Thanks for your reply..
